api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -2120,7 +2120,7 @@ def configure_nozzle_layout(  # noqa: C901
 
     @requires_version(2, 20)
     def detect_liquid_presence(self, well: labware.Well) -> bool:
-        """Check if there is liquid in a well.
+        """Checks if there is liquid in a well. Will not raise an error if it does not detect liquid. Flex pipettes only. See :ref:`lpd` and :ref:`detect-liquid-presence`.
```

Comment:
```suggestion
        """Checks if there is liquid in a well. 
        
        Returns ``True`` if liquid is present and ``False`` if liquid is not present. Will not raise an error if it does not detect liquid. Flex pipettes only. See :ref:`detect-liquid-presence`.
```
First sentence of a docstring should stand alone.

Removed xref to global detection, which is more like `require_liquid_presence()`.